src/components/list/themes/light/list-item.indigo.scss

```diff
@@ -1,23 +1,19 @@
+@use './list-item.base.scss';
```

Comment:
This should rather be imported and used as the default static styles on the list-header.ts. It shouldn't be imported here.